### ISSUE TITLE
BUG: cannot create experiment

### ISSUE THREAD

**Issue by qai222** [neptune-ai/neptune-client#671]
### Describe the bug
`neptune.create_experiment()` errors out for `neptune-client==0.10.6`, but works fine with `neptune-client==0.9.8`

### Reproduction
```
import neptune
neptune.init(api_token=neptune_api_token, project_qualified_name=neptune_proj_name)
experiment = neptune.create_experiment()
```
while it creates an experiment in the project (can be seen from front-end), it gives the following error 

```
----InternalClientError-----------------------------------------------------------------------

Neptune Client Library encountered an unexpected Internal Error:
Unexpected response from server: b'<!doctype html>\n<html>\n<head>\n  <!-- neptune-frontend version: frontend/3.7.9 -->\n  <meta charset="utf-8">\n  <title>neptune.ai</title>\n  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">\n  <meta name="description" content="">\n  <meta name="viewport" content="width=1280, initial-scale=1">\n  <link href=\'https://fonts.googleapis.com/css?family=Source+Sans+Pro\' rel=\'stylesheet\' type=\'text/css\'>\n  <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5">\n  <script src="https://js.stripe.com/v3/"></script>\n  <script type="text/javascript" src="/docker-config.js?v=frontend/3.7.9"></script>\n  <script type="text/javascript" src="/commons.js?v=frontend/3.7.9"></script>\n  <script type="text/javascript" src="/ergane.js?v=frontend/3.7.9"></script>\n</head>\n<body>\n  <div id="app-viewport">\n    <div id="maintenance-app"></div>\n    <div id="app"></div>\n  </div>\n  <script>__REACT_DEVTOOLS_GLOBAL_HOOK__ = parent.__REACT_DEVTOOLS_GLOBAL_HOOK__</script>\n  <script type="text/javascript" src="/app.js?v=frontend/3.7.9"></script>\n</body>\n</html>\n'

Please contact Neptune support
```

### Expected behavior
no error should be raised

### Traceback
If applicable, add traceback or log output/screenshots to help explain your problem.
### Environment
**The output of `pip list`:**
```
attrs                  21.2.0
bravado                11.0.3
bravado-core           5.17.0
certifi                2021.5.30
charset-normalizer     2.0.4
click                  8.0.1
colorama               0.4.4
future                 0.18.2
gitdb                  4.0.7
GitPython              3.1.18
idna                   3.2
jsonpointer            2.1
jsonref                0.2
jsonschema             3.2.0
monotonic              1.6
msgpack                1.0.2
neptune-client         0.10.6
numpy                  1.21.2
oauthlib               3.1.1
packaging              21.0
pandas                 1.3.2
Pillow                 8.3.1
pip                    21.0.1
PyJWT                  2.1.0
pyparsing              2.4.7
pyrsistent             0.18.0
python-dateutil        2.8.2
pytz                   2021.1
PyYAML                 5.4.1
requests               2.26.0
requests-oauthlib      1.3.0
rfc3987                1.3.8
setuptools             52.0.0.post20210125
simplejson             3.17.5
six                    1.16.0
smmap                  4.0.0
strict-rfc3339         0.7
swagger-spec-validator 2.7.3
typing-extensions      3.10.0.0
urllib3                1.26.6
webcolors              1.11.1
websocket-client       1.2.1
wheel                  0.37.0
wincertstore           0.2
```
**The operating system you're using:** win 10
**The output of `python --version`:** 3.8.11

**Comment by HubertJaworski** [comment]
Hi @qai222 

Thanks for letting us know!

0.10.6 was a bugfix release and looks like another bug managed to slip in.

I just tested for regression and 0.10.5 should work fine in your setup.

You can install it using: `pip install neptune-client==0.10.5`

I'll let the team know and we'll jump right into fixing it.

Best,
Hubert

**Comment by HubertJaworski** [comment]
Hi @qai222 ,

We just released #672 with `neptune-client==0.10.7`

I tested it on a Windows machine and it solves your bug.

I'm closing this, but let us know in case something won't work for you

Best,
Hubert
